subprojects/common/throwable-utils/src/main/kotlin/com/avito/android/util/StackTraceFormatter.kt

```diff
@@ -4,9 +4,9 @@ import org.apache.commons.text.TextStringBuilder
 import java.io.PrintWriter
 
 /**
- * Prepare stacktrace for report
+ * Get stacktrace as list of lines from throwable
```

Comment:
In Kotlin 1.4 we can use `Throwable.stackTraceToString()` instead of this function.